packages/jest-cli/src/cli/index.js

```diff
@@ -67,6 +68,24 @@ const runCLI = async (
       outputStream,
     );
 
+    if (argv.clearCache) {
+      let clearCacheError;
+
+      configs.map(config => {
+        rimraf(config.cacheDirectory, [], () => {
```

Comment:
I'm afraid this is completely wrong and doesn't work at all. 
Please refer to `rimraf` tests on how to use it:
https://github.com/isaacs/rimraf/blob/d84fe2cc6646d30a401baadcee22ae105a2d4909/test/basic.js#L48-L55.

Also I see no point in using async API, since we're not doing anything else in this time. Please use `rimraf.sync`, this way you'll avoid temporary `clearCacheError` variable and perf overhead of async call.

Reply:
Seemed to work when I tested it, possible I was wrong. Will switch to sync.